kura/distrib/src/main/ant/build_equinox_distrib.xml

```diff
@@ -1315,6 +1326,17 @@ fi]]>
 		</propertyfile>
 	</target>
 
+	<target name="linux-snapctl-config" if="is.snapctl" depends="linux-config">
+		<propertyfile file="${project.build.directory}/${build.output.name}/config.ini">
+			<entry key="osgi.bundles" operation="+"
+					value=", reference:file:${kura.install.dir}/${kura.symlink}/${plugins.folder}/org.eclipse.kura.linux.snapctl.provider_${org.eclipse.kura.linux.snapctl.provider.version}.jar@4" />
+			<entry key="osgi.bundles" operation="+"
```

Comment:
Sorry, probably I've missed the point: the emulator bundles should be provided only in NN profiles, but in the is.snapctl you set that has to be not NN. Right?

Reply:
@MMaiero this is attempt to add only one additional target. Like supper set bundle, so it has emulator and real network providers, and correct one is chosen when snap is built by editing config file.
But I agree that this was created bundle is confusing.
@MMaiero up to you to guide me what's better approach, add one ubuntu-snap target, or add two targets `ubuntu-snap` `ubuntu-snap-nn`

Reply:
If that is not a problem for you, I'll prefer to have two targets. In this way it keeps also the consistency with the other targets.

Reply:
@MMaiero no problem. Then next question is if you are OK to have folded into one target differentiations architectures? This is related to comment below about tinyb
So `ubuntu-snap/ubuntu-snap-nn` will have libraries for all architectures, this is mostly related to tinyb one.
Otherwise we will need following targets:
```
ubuntu-snap-armhf
ubuntu-snap-arm64
ubuntu-snap-amf64
ubuntu-snap-nn-armhf
ubuntu-snap-nn-arm64
ubuntu-snap-nn-amf64
```
You have to remember that snaps support all different architectures, I have skipped here 32b x86, as you do not seem to have any support for it